aspnetcore/tutorials/dotnet-watch.md

```diff
@@ -80,7 +86,7 @@ Fix the bug in the `Product` method of *MathController.cs* so it returns the pro
 ```csharp
 public static int Product(int a, int b)
 {
-  return a * b;
+    return a * b;
```

Comment:
It looks like 4 spaces are used in all other code snippets and in the sample projects, so I let my OCD go wild here as I'm in the file anyway. :)